src/js/features/beacon/beaconConfig.store.ts

```diff
@@ -7,7 +7,7 @@ import { printAPIError } from '@/utils/error.util';
 export const getBeaconConfig = createAsyncThunk<BeaconConfigResponse, void, { state: RootState; rejectValue: string }>(
   'beaconConfig/getBeaconConfig',
   (_, { getState, rejectWithValue }) => {
-    const beaconInfoEndpoint = getState()?.config?.beaconUrl + '/info';
+    const beaconInfoEndpoint = getState()?.config?.beaconUrl + '/overview';
```

Comment:
we should probably reject this if beaconUrl is undefined, but since the behaviour is already like this, this PR is good to merge

Reply:
The UI isn't rendered until beaconUrl is defined, so it would be hard to get into this state.

Reply:
it's more that having the question marks sort of implies that it can somehow

also it's unneeded to have a `?` after `getState()` and `config` I think anyway, since getState.config should always return an object...

Reply:
This is probably a code copying crime on my part.